pkg/apis/core/validation/controllerregistration.go

```diff
@@ -58,8 +57,8 @@ var SupportedExtensionKinds = sets.NewString(
 	extensionsv1alpha1.NetworkResource,
 	extensionsv1alpha1.OperatingSystemConfigResource,
 	extensionsv1alpha1.WorkerResource,
-	// TODO: drop this, once we have externalized DNSProvider functionality, see https://github.com/gardener/gardener/issues/5270
-	dnsv1alpha1.DNSProviderKind,
+	// TODO: drop this with v1.55 or later to still support removal of any controller registrations specifying resources of kind DNSProvider, see https://github.com/gardener/gardener/issues/5270
+	"DNSProvider",
```

Comment:
Can you drop `DNS{Provider,Entry,Onwer}` / `dns.gardener.cloud` references also from the following places?

https://github.com/gardener/gardener/blob/b182557e8acd4ffc4e59a4bf324e070c35c9907c/pkg/apis/core/types_controllerregistration.go#L48-L50
https://github.com/gardener/gardener/blob/b182557e8acd4ffc4e59a4bf324e070c35c9907c/pkg/apis/core/types_controllerregistration.go#L55-L57
https://github.com/gardener/gardener/blob/master/docs/extensions/dnsrecord.md
https://github.com/gardener/gardener/blob/master/docs/usage/trouble_shooting_guide.md
https://github.com/gardener/gardener/blob/b182557e8acd4ffc4e59a4bf324e070c35c9907c/test/framework/shootmigrationtest.go#L396

Reply:
done